src/App.tsx

```diff
@@ -218,24 +228,16 @@ const Navigation = React.memo(() => {
   );
 });
 
-const Logo = React.memo(() => {
+const Logo = React.memo(function Logo() {
```

Comment:
It helped with figuring out exactly where this was rendered. I was using the react devtools to inspect the component structure and if you use React.memo, your component shows as `Anonymous` rather than it's actual name making it impossible to find. You can either add a name to your function like this or do `Logo.displayName = 'Logo'` before you export it